### PR TITLE
PUB-1663 Clear Subscription

### PR DESCRIPTION
**Pull request by junaidiqbalmoj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-1663

### Change description ###

Add email template to send email to subscribers when their subscription is being deleted by System Admin as part of Delete court journey.

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotificationController.java [hunk 1]
```diff
@@ -213,4 +218,19 @@ public ResponseEntity<String> sendSystemAdminUpdate(@RequestBody SystemAdminActi
             notificationService.sendSystemAdminUpdateEmailRequest(body)
         ));
     }
+
+    @ApiResponse(responseCode = OK_RESPONSE, description = "Unidentified blob email "
```
Comment (ChrisS1512) [inline src/main/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotificationController.java:222]: OK response message needs updating

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotificationController.java [hunk 2]
```diff
@@ -213,4 +218,19 @@ public ResponseEntity<String> sendSystemAdminUpdate(@RequestBody SystemAdminActi
...
+        + "successfully sent with referenceId: {Id}")
+    @ApiResponse(responseCode = BAD_REQUEST, description = BAD_PAYLOAD_ERROR_MESSAGE)
+    @ApiResponse(responseCode = BAD_REQUEST, description = NOTIFY_EXCEPTION_ERROR_MESSAGE)
+    @ApiResponse(responseCode = AUTH_RESPONSE, description = NOT_AUTHORIZED_MESSAGE)
+    @Operation(summary = "Send the location subscription deletion email to all the subscribers")
+    @PostMapping("/location-subscription-delete")
+    public ResponseEntity<String> sendDeleteLocationSubscriptionEmail(
```
Comment (ChrisS1512) [inline src/main/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotificationController.java:229]: I would update this to be generic, maybe just deleteLocation, as we also include the artefact deletion too in this template (I think)

Comment (ChrisS1512) [reply src/main/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotificationController.java:229]: (Also similarly other classes will renaming too to be generic)

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotificationController.java (hunk 1 above)
Comment (junaidiqbalmoj) [reply src/main/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotificationController.java:222]: thanks, updated.

**Review by ChrisS1512 — APPROVED** [review]

**Review by KianKwa — APPROVED** [review]

**Pull request merged**